bots/tests-invoke

```diff
@@ -166,9 +166,9 @@ class PullTask(object):
             traceback.print_exc()
             return "Rebase failed"
 
-        # COMPAT: If the bots directory doesn't exist in this branch, check it out from master
+        # Always use bots/ and test images from master
         try:
-            if subprocess.call([ "git", "ls-tree", "-d", "HEAD:bots/"], stdout=DEVNULL) != 0:
+            if self.base != "master":
```

Comment:
I think that if self.base is empty/null, it is often the case that we're on "master". I don't think we should checkout in those cases.

Either that, or you can get more fancy and see if we're a descendant of master. But that may be too complex here. Up to you.